tutorials/get-started-canonical-kubernetes/get-started-canonical-kubernetes.md

```diff
@@ -1,187 +1,179 @@
+summary: Learn how to operate a production-ready Kubernetes cluster. Kubernetes is a great open-source orchestration system for cloud native infrastructure.
+categories: jaas
 tags: jaas, juju, kubernetes, beginner, container, docker, cloud
-difficulty: 2
+difficulty: 3
 status: Published
 Published: 2017-06-01
- 
+
 ---
- 
- 
+
+
 # Get started with Canonical Kubernetes
- 
- 
+
+
 ## Overview
 Duration: 1:00
- 
+
 ### About the Canonical Distribution of Kubernetes
- 
-This solution is a highly available (HA) production ready Kubernetes cluster — with 2 masters, 3 workers, 3 etcd nodes and a load balancer for the HA control plane. It includes logging, monitoring, and the operational tools to automate deployment, scaling, and lifecycle management of your cluster. 
- 
+
+This solution builds a highly available (HA) production ready Kubernetes cluster — with 2 masters, 3 workers, 3 etcd nodes and a load balancer for the HA control plane. It includes logging, monitoring, and the operational tools to automate deployment, scaling, and lifecycle management of your cluster.
+
 Learn more about [The Canonical Distribution of Kubernetes](https://jujucharms.com/canonical-kubernetes/).
 Learn more about [Kubernetes](https://kubernetes.io/).
- 
+
 ![Kubernetes bundle](./images/kubernetes-bundle.png)
- 
- 
+
+
 ### In this tutorial you’ll learn how to...
- 
-- Get your Kubernetes cluster up and running.
-- Open the K8s dashboard.
-- Control your cluster from the kubectl CLI client.
-- Deploy your first container workload.
-- Add extra features to your Kubernetes cluster.
- 
- 
+
+- Get your Kubernetes cluster up and running
+- Open the K8s dashboard
+- Control your cluster from the kubectl CLI client
+- Deploy your first container workload
+- Add extra features to your Kubernetes cluster
+
+
 ### You will need...
- 
-* An [Ubuntu One account](https://login.ubuntu.com/).
-* An SSH key in your JAAS account. If you haven’t done this already, [learn how to add your SSH key](https://jujucharms.com/docs/2.1/users-auth#credentials-and-ssh-keys).
-* Basic knowledge of command line.
- 
- 
- 
+
+* An [Ubuntu One account](https://login.ubuntu.com/)
+* An SSH key. [More on using SSH keys in Juju](https://jujucharms.com/docs/2.1/users-auth#credentials-and-ssh-keys).
+* Basic knowledge of command line
+
+
+
 ## Deploying your model with JAAS
 Duration: 1:00
- 
-With JAAS, you don’t need to deal with the setup and configuration of Juju. All you will need is your preferred cloud credentials.
- 
-You will need to deploy [The Canonical Distribution of Kubernetes](https://jujucharms.com/new?dd=bundle/canonical-kubernetes) before you get started. 
- 
+
+With JAAS, you don’t need to deal with the setup and configuration of Juju. All you will need is credentials for AWS, GCE, or Azure.
+
+Kick off deployment of your Kubernetes cluster by clicking the follow link: [The Canonical Distribution of Kubernetes](https://jujucharms.com/new?dd=bundle/canonical-kubernetes).
```

Comment:
To kick off, [deploy The Canonical Distribution of Kubernetes] (https://jujucharms.com/new?dd=bundle/canonical-kubernetes) (if you haven't done it already)